apps/omg_watcher/test/omg_watcher/db/repo_test.exs

```diff
@@ -0,0 +1,44 @@
+# Copyright 2019 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.Watcher.DB.RepoTest do
+  import Mox
+
+  use ExUnitFixtures
+  use OMG.Watcher.Case, async: true
+  use OMG.Fixtures
+
+  import Ecto.Query, only: [from: 2]
+
+  alias OMG.Watcher.DB
+
+  # Make sure mocks are verified when the test exits
+  setup :verify_on_exit!
+
+  @tag fixtures: [:phoenix_ecto_sandbox]
```

Comment:
to be honest, I'm not sure why. i was trying to just use the `watcher` fixture as I don't need all the extra stuff, but i kept seeing 'db unavailable` type errors until i used `phoenix_ecto_sandbox`. happy to update this to whatever works best though.

Reply:
I just discovered why I needed this know. Our fixtures need some love. If I run `MIX_ENV=test mix test test/omg_watcher/db/repo_test.exs` and run it as a single test `phoenix_ecto_sandbox` is needed.

If I run something like `MIX_ENV=test mix test test/omg_watcher` then this will cause a failure  `Failed to start Ranch listener OMG.WatcherRPC.Web.Endpoint.HTTP in :ranch_tcp:listen([port: 7435]) for reason :eaddrinuse (address already in use)`

The fixture required by the test is dependent on how the test is run which is likely not what we want, but beyond the scope of this PR.